src/App.jsx

```diff
@@ -1,27 +1,28 @@
-import React, { useState } from "react";
+import React from "react";
+import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
+
 import Navbar from "./components/Navbar";
-import TaskList from "./components/TaskList";
 import LandingPage from "./components/LandingPage";
+import HomePage from "./pages/HomePage";
+import AddEditPage from "./pages/AddEditPage";
+
 import "./App.css";
 
 function App() {
-  const [started, setStarted] = useState(false);
-
-  const handleGetStarted = () => {
-    setStarted(true);
-  };
-
   return (
-    <div>
-      <Navbar />
-      <div className="App">
-        {!started ? (
-          <LandingPage onGetStarted={handleGetStarted} />
-        ) : (
-          <TaskList />
-        )}
+    <Router>
+      <div>
+        <Navbar />
+        <div className="App">
+          <Switch>
+            <Route path="/" exact component={LandingPage} />
+            <Route path="/home" component={HomePage} />
+            <Route path="/add-task" exact component={AddEditPage} />
+            <Route path="/edit-task/:taskId" component={AddEditPage} /> {/* Route for editing tasks */}
```

Comment:
Try avoiding comments as much as posible

```suggestion
            <Route path="/edit-task/:taskId" component={AddEditPage} /> 
```